docs/articles/life-cycle/setup-and-teardown.rst

```diff
@@ -0,0 +1,289 @@
+Setup and Teardown Hooks
+########################
+
+Pytest-mh provides multiple setup and teardown hooks that you can use to setup
+the test environment and later revert all changes that were done during the
+setup and testing.
+
+It is possible to setup and teardown individual hosts, topologies, roles and
+utilities. The scope of individual hooks spans from a whole pytest session
+(called only once per session), topology (called once per multihost topology)
```

Comment:
Can I change `called only once per session` by `called only once per run`? I think it's clearer

Reply:
Thanks for the explanation. Then, let's keep it as it is.